src/main/scala/com/vngrs/etl/Pipeline.scala

```diff
@@ -0,0 +1,94 @@
+package com.vngrs.etl
+
+import com.vngrs.etl.combiners.{Union, Zip}
+import org.apache.spark.SparkContext
+import org.apache.spark.rdd.RDD
+
+import scala.reflect.ClassTag
+
+/**
+  * Represents ETL Pipeline.
+  *
+  * @param rdd The [[org.apache.spark.rdd.RDD]] which will be ETLed
+  * @tparam A Type of the [[org.apache.spark.rdd.RDD]]
+  */
+final class Pipeline[A: ClassTag](val rdd: RDD[A]) {
+
+  /**
+    * Transforms [[com.vngrs.etl.Pipeline]] with given [[com.vngrs.etl.Transform]].
+    *
+    * @param transformer [[com.vngrs.etl.Transform]]
+    * @tparam B Type of the output [[com.vngrs.etl.Pipeline]]
+    * @return Transformed [[com.vngrs.etl.Pipeline]]
+    */
+  def transform[B: ClassTag](transformer: Transform[A, B]): Pipeline[B] = Pipeline(transformer(rdd))
+
+  /**
+    * Loads this [[com.vngrs.etl.Pipeline]] to given [[com.vngrs.etl.Load]](s).
+    *
+    * @param loaders [[com.vngrs.etl.Load]]
+    */
+  def load(loaders: Load[A]*): Unit = loaders.foreach(loader => loader(rdd))
```

Comment:
I didn't understand how this works. Can you explain for me how this loading works wen you want to load, let's say three sources?